src/features/customerView/queries.ts

```diff
@@ -76,6 +76,25 @@ export const INDIVIDUAL_CUSTOMER_QUERY = gql`
                 }
               }
             }
+            order {
```

Comment:
Apollo sometimes causes weird issues with `id`s are not queried. 
```suggestion
            order {
```

Reply:
Will add `id`s.